core/modules/user/user.module

```diff
@@ -323,14 +323,12 @@ function user_load_by_name($name) {
  * Save a new or updated user.
  *
  * @param $account
```

Comment:
Sorry for being so picky... The datatype should get added. Actually I had to look up the `@param` documentation, too. :wink: 

> `@param datatype $paramname`
> ... The datatype should be a valid PHP type (int, string, bool, etc), a class name for the type of object, or simply "mixed".

So in our case that should be like:

`@param User $account`

As "User" is the name of the class. I hope, I don't overstretch your patience. Your work is really appreciated. :+1:

Reply:
> I hope, I don't overstretch your patience. Your work is really appreciated. +1
@indigoxela On the contrary I do appreciate your pointers as I'm learning from you a lot! When it comes to making it right, I'm nothing but patience.